pandas/core/reshape/pivot.py

```diff
@@ -612,7 +612,7 @@ def crosstab(
         margins=margins,
         margins_name=margins_name,
         dropna=dropna,
-        **kwargs,
+        **kwargs,  # type: ignore[arg-type]
```

Comment:
don't add type ignores

Reply:
okay I'll skip pivot in this PR (this is a weird issue with typing `**kwargs`)